vhdlweb.wsgi

```diff
@@ -84,6 +101,11 @@ startercode + \
 </script>
 
 <button onClick="compile()">Build!</button>
+
+<a id = "save">
+<button  onClick="saveFile()"> saveFile </button>
```

Comment:
Can we label the button "download copy" or "save locally"?  I just want to differentiate from "save on the server" or "save in my web browser so it's here when I come back".